File: app/src/main/java/com/mesada/shop/helper/RandomNumber.java
package com.mesada.shop.helper;

import java.util.Random;

public class RandomNumber {

    public static long generateID() {
        Random rnd = new Random();
        char [] digits = new char[11];
        digits[0] = (char) (rnd.nextInt(9) + '1');
        for(int i=1; i<digits.length; i++) {
            digits[i] = (char) (rnd.nextInt(10) + '0');
        }
        return Long.parseLong(new String(digits));
    }

}

File: app/src/main/java/com/mesada/shop/helper/RandomNumber.kt
package com.mesada.shop.helper

import java.util.*

object RandomNumber {
    fun generateID(): Long {
        val rnd = Random()
        val digits = CharArray(11)
        digits[0] = (rnd.nextInt(9) + '1'.toInt()).toChar()
        for (i in 1 until digits.size) {
            digits[i] = (rnd.nextInt(10) + '0'.toInt()).toChar()
        }
        return String(digits).toLong()
    }
}